src/stylesheets/abstracts/_variables.scss

```diff
@@ -17,18 +17,23 @@ $color-dark-purple: #252224; //text
 
 /*--------- Fonts ---------*/
 
+
 @font-face {
   font-family: 'Geomanist';
-  src: url(../src/fonts/geomanist/Geomanist-Regular.otf);
+  font-weight: 400;
+  font-style: normal;
+  src: url(../src/fonts/geomanist-regular-webfont.woff2);
 }
 
 @font-face {
-  font-family: 'Geomanist-Medium';
-  src: url(../src/fonts/geomanist/Geomanist-Medium.otf);
+  font-family: 'Geomanist';
+  font-weight: 500;
```

Comment:
Could be `font-weight: bold`. I just find the words easier than the numbers, unless I'm dealing with multiple weights. :shrug: